content/docs/latest/provisioning/ignition/boot-process.md

```diff
@@ -111,6 +111,22 @@ Here is an example config with an additional `links` entry that ensures that the
 }
 ```
 
+#### Debugging Ignition
+
+In the event of Ignition failure, there are most of the time three scenarios:
+* Instance console access with emergency shell
```

Comment:
```suggestion
* **Instance console access with emergency shell:**
```
The formatting looks off in the rendered website. I think we can try this.

Reply:
or use `<br>`

Reply:
or make it bold plus the `<br>`, up to you ;)

Reply:
(With `<br>` one can avoid starting a new paragraph - sometimes markdown is a bit limited and it makes sense to resort to html elements)

Reply:
(Some markdown implementations also allow to add `\`  at the end of the line to have a similar effect)